addons/playersuppression/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,13 @@
+class Extended_PreInit_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_FILE(XEH_preInit));
+    };
+};
+
+class Extended_Fired_EventHandlers {
+    class All {
+        class ADDON {
+            clientFired = QUOTE(if ((GVAR(enablePlayerSuppression) && !isDedicated)) then {_this call FUNC(fired)});
```

Comment:
Didn't know about the firedBIS thingy, thanks!
I actually wanted to use !isDedicated since it simply shouldn't be executed on a dedicated server. Using isServer doesn't work because the hoster wouldn't get the effects when hosting a local session.